module/Finna/src/Finna/AjaxHandler/GetDescription.php

```diff
@@ -183,17 +183,19 @@ public function handleRequest(Params $params)
                     return $this->formatResponse(['html' => $content]);
                 }
             }
-            $language = $this->translator->getLocale();
-            if ($summary = $driver->getSummary($language)) {
-                $summary = implode("\n\n", $summary);
+            if (!($driver instanceof \Finna\RecordDriver\SolrLido)) {
```

Comment:
Laita tähän kommentti, että `// For LIDO records the summary is displayed separately from description in the core template` niin ei tartte myöhemmin ihmetellä, mikä tämä tarkistus on.